datalad/runner/tests/test_nonasyncrunner.py

```diff
@@ -582,7 +563,9 @@ def timeout(self, fd: Optional[int]) -> bool:
 def test_exit_3() -> None:
     # Expect the process to be closed after
     # the generator exits.
-    rt = ThreadedRunner(cmd=["sleep", "4"],
+    rt = ThreadedRunner(cmd=["waitfor", "/T", "4", "TheComputerTurnsIntoATulip"]
+                        if on_windows
```

Comment:
FTR: seems to not relate to mixin's RFing but since just in the tests, I guess it is ok to not bother describing in the changelog